tests/drivers/gpio/gpio_basic_api/boards/rpi_pico2_rp2350a_m33.conf

```diff
@@ -0,0 +1 @@
+CONFIG_SKIP_PULL_TEST=y
```

Comment:
@henrikbrixandersen 

Let me confirm one thing.
Currently, Pico2 has an errata that causes incorrect behavior when GPIO_INPUT and GPIO_PULL_DOWN are specified at the same time. For this reason, it specifies SKIP_PULL here to avoid testing this case, but is this solution okay?

Reply:
Yes.

Reply:
Thank you for your reply. I'll move forward with this.